index.html

```diff
@@ -350,66 +350,93 @@ <h3>Use Cases and Requirements</h3>
 specifically:
         </p>
 
-        <ul>
+        <ol>
           <li>
-<a>Holders</a> receive and store <a>verifiable credentials</a> from
-<a>issuers</a>. <a>Holders</a> might interact with an <a>issuer</a> through an
-agent that the <a>issuer</a> does not need to trust.
+<a>Verifiable Credentials</a> represent statements made by an <a>Issuer</a> in
+a tamper evident and privacy respecting manner.
           </li>
           <li>
-<a>Holders</a> are positioned between <a>issuers</a> and <a>verifiers</a>
-and use <a>verifiable credentials</a> to make <a>verifiable presentations</a>
-to <a>verifiers</a>.
+<a>Holders</a> assemble collections of claims from different issuers into a
+single artifact, a <a>Verifiable Presentation</a>.
           </li>
           <li>
-<a>Holders</a> might interact with <a>verifiers</a> through an agent that
-<a>verifiers</a> do not need to trust; <a>verifiers</a> only need to trust
```

Comment:
again, the trust between issuer/verifier only needs to exist. No trust between issuer/holder needs to happen. I may have missed where this point is made on the right.